website/content/docs/plugins/index.mdx

```diff
@@ -0,0 +1,200 @@
+---
+description: |
+  Packer Plugins allow new functionality to be added to Packer without modifying
+  the core source code. Packer plugins are able to add new builders,
+  provisioners, hooks, and more.
+page_title: Plugins
+sidebar_title: Packer Plugins
+---
+
+# Packer Plugins
+
+Packer Plugins allow new functionality to be added to Packer without modifying
+the core source code. Packer plugins are able to add new builders, provisioners,
+hooks, and more. In fact, much of Packer itself is implemented by writing
+plugins that are simply distributed with Packer. For example, many of the
+builders, provisioners, and more that ship with Packer are implemented as
+Plugins that are simply hardcoded to load with Packer.
+
+* This page documents how to install plugins.
+* Plugins cannot be configured yet.
+* If you're interested in developing plugins, see the [developing 
+plugins](/docs/plugins/creation#developing-plugins) page.
+
+The current official listing of available Packer plugins can be found
+[here](/community-tools#third-party-plugins). This is an incomplete list, and
```

Comment:
These plugins will work only if they're manually installed. Should we suggest the list here now?

Reply:
yes !